src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
If they were always used together you could put them in a single module and reference them from an object it returned. However that is probably less compressible and slower since the property name won't be minified the way a variable is?